`ensure-graalvm` and `ensure-java` in qDup not working

From this change: https://github.com/quarkusio/spring-quarkus-perf-comparison/pull/268/changes#diff-4474f2b3348d3d73e9b4bf328026bfe5a6d04cd2d856c54d0240a2d1b9113fe3

the `ensure-java` and `ensure-graalvm` scripts are no longer working when specifying java versions without paths.

```
07:34:44,675 [ test-runner:setup-838 @ target-host ] script-cmd: ensure-graalvm
07:34:44,676 [ test-runner:setup-838 @ target-host ] ensure-graalvm
07:34:44,678 [ test-runner:setup-838 @ target-host ] read-state: 
07:34:44,679 [ test-runner:setup-838 @ target-host ] Showing GraalVM version
07:34:45,156 [ test-runner:setup-838 @ target-host ] $GRAALVM_HOME/bin/java -version
openjdk version "21.0.10" 2026-01-20
OpenJDK Runtime Environment (Red_Hat-21.0.10.0.7-2) (build 21.0.10+7)
OpenJDK 64-Bit Server VM (Red_Hat-21.0.10.0.7-2) (build 21.0.10+7, mixed mode, sharing)
```

and then this causes things to not use the correct java versions, and native compilation is failing:

`[ERROR] Failed to execute goal org.graalvm.buildtools:native-maven-plugin:0.10.6:compile (default-cli) on project springboot3: native-image is not installed in your JAVA_HOME.This probably means that the JDK at '/home/jenkins/.sdkman/candidates/java/current' is not a GraalVM distribution. The GraalVM Native Maven Plugin requires GRAALVM_HOME or JAVA_HOME to be a GraalVM distribution. -> [Help 1]`

--- a/.gitignore
+++ b/.gitignore
@@ -17,3 +17,4 @@ buildNumber.properties
 .classpath
 /.idea/
 /.profileconfig.json
+.claude

--- a/scripts/perf-lab/helpers/java.yml
+++ b/scripts/perf-lab/helpers/java.yml
@@ -1,7 +1,7 @@
 name: Java related scripts
 scripts:
   ensure-graalvm:
-    - read-state: ${{config.jvm.graalvm.home}}
+    - sh: echo "${{config.jvm.graalvm.home:}}"
       then:
         - regex: ^$
           then:
@@ -30,7 +30,7 @@ scripts:
     - sh: $GRAALVM_HOME/bin/java -version
 
   ensure-java:
-    - read-state: ${{config.jvm.home}}
+    - sh: echo "${{config.jvm.home:}}"
       then:
         - regex: ^$
           then:

